net/makuosan/Portfile

```diff
@@ -1,15 +1,16 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem              1.0
+PortGroup               openssl 1.0
+
+openssl.branch          1.1
 
 name                    makuosan
-version                 1.2.1
-revision        4
-homepage                http://lab.klab.org/wiki/Makuosan
+version                 1.3.6
+revision                0
+homepage                https://sourceforge.net/projects/makuosan
 categories              net
-platforms               darwin
 maintainers             nomaintainer
-use_parallel_build      yes
 
```

Comment:
`port lint` complains about a missing `license` field; please add that, from upstreams sourceforge site it looks like it's `GPL3` but please verify from the source files.

Reply:
Thank you for noticing, will fix it.